recycleBitmaps throw NullPointerException while onDestroy after LottieAnimationView updateBitmap with a null bitmap

LottieAnimationView throws NullPointerException while activity onDestoy. Because a bitmap in ImageAssetManager bitmaps is null.  I call LottieAnimationView updateBitmap(id, null) because I need clear the bitmap, after this the exception would occur while onDestroy.

`Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.graphics.Bitmap.recycle()' on a null object reference
                                                                                 at com.airbnb.lottie.manager.ImageAssetManager.recycleBitmaps(ImageAssetManager.java:96)
                                                                                 at com.airbnb.lottie.LottieDrawable.recycleBitmaps(LottieDrawable.java:142)
                                                                                 at com.airbnb.lottie.LottieAnimationView.recycleBitmaps(LottieAnimationView.java:277)
                                                                                 at com.airbnb.lottie.LottieAnimationView.onDetachedFromWindow(LottieAnimationView.java:269)
                                                                                 at android.view.View.dispatchDetachedFromWindow(View.java:15668)
                                                                                 at android.view.ViewGroup.dispatchDetachedFromWindow(ViewGroup.java:3183)
                                                                                 at android.view.ViewGroup.dispatchDetachedFromWindow(ViewGroup.java:3183)
                                                                                 at android.view.ViewGroup.removeViewInternal(ViewGroup.java:4719)
                                                                                 at android.view.ViewGroup.removeViewInternal(ViewGroup.java:4693)
                                                                                 at android.view.ViewGroup.removeView(ViewGroup.java:4624)
                                                                                 at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1205)
                                                                                 at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1286)
                                                                                 at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1268)
                                                                                 at android.support.v4.app.FragmentManagerImpl.dispatchDestroy(FragmentManager.java:2180)
                                                                                 at android.support.v4.app.FragmentController.dispatchDestroy(FragmentController.java:271)
                                                                                 at android.support.v4.app.FragmentActivity.onDestroy(FragmentActivity.java:386)
                                                                                 at android.support.v7.app.AppCompatActivity.onDestroy(AppCompatActivity.java:202)`

--- a/lottie/src/main/java/com/airbnb/lottie/manager/ImageAssetManager.java
+++ b/lottie/src/main/java/com/airbnb/lottie/manager/ImageAssetManager.java
@@ -66,25 +66,24 @@ public class ImageAssetManager {
   }
 
   @Nullable public Bitmap bitmapForId(String id) {
-    Bitmap bitmap = imageAssets.get(id).getBitmap();
+    LottieImageAsset asset = imageAssets.get(id);
+    if (asset == null) {
+      return null;
+    }
+    Bitmap bitmap = asset.getBitmap();
     if (bitmap != null) {
       return bitmap;
     }
 
-    LottieImageAsset imageAsset = imageAssets.get(id);
-    if (imageAsset == null) {
-      return null;
-    }
-
     if (delegate != null) {
-      bitmap = delegate.fetchBitmap(imageAsset);
+      bitmap = delegate.fetchBitmap(asset);
       if (bitmap != null) {
         putBitmap(id, bitmap);
       }
       return bitmap;
     }
 
-    String filename = imageAsset.getFileName();
+    String filename = asset.getFileName();
     BitmapFactory.Options opts = new BitmapFactory.Options();
     opts.inScaled = true;
     opts.inDensity = 160;
@@ -122,7 +121,10 @@ public class ImageAssetManager {
       Iterator<Map.Entry<String, LottieImageAsset>> it = imageAssets.entrySet().iterator();
       while (it.hasNext()) {
         Map.Entry<String, LottieImageAsset> entry = it.next();
-        entry.getValue().getBitmap().recycle();
+        Bitmap bitmap = entry.getValue().getBitmap();
+        if (bitmap != null) {
+          bitmap.recycle();
+        }
         it.remove();
       }
     }

